charts/descheduler/templates/clusterrole.yaml

```diff
@@ -21,4 +21,11 @@ rules:
 - apiGroups: ["scheduling.k8s.io"]
   resources: ["priorityclasses"]
   verbs: ["get", "watch", "list"]
+{{- if .Values.rbac.podSecurityPolicy.create }}
```

Comment:
@invidian what do you think about instead using `.Values.podSecurityPolicy.create` instead of `.Values.rbac.podSecurityPolicy.create`. I feel like the `PodSecurityPolicy` is separate from RBAC settings.

Reply:
Sounds good. Some charts put it in `rbac`, but I agree it's rather separate from RBAC :+1: